plugins/keycloak-backend/README.md

```diff
@@ -55,93 +55,98 @@ When using client credentials, the access type must be set to `confidential` and
-
-        export default async function createPlugin(
-          env: PluginEnvironment,
-        ): Promise<Router> {
-          const builder = await CatalogBuilder.create(env);
-
-      +   builder.addEntityProvider(
-      +     KeycloakOrgEntityProvider.fromConfig(env.config, {
-      +       id: 'development',
-      +       logger: env.logger,
-      +       schedule: env.scheduler.createScheduledTaskRunner({
-      +         frequency: { minutes: 1 },
-      +         timeout: { minutes: 1 },
-      +         initialDelay: { seconds: 15 }
-      +       }),
-      +     }),
-      +   )
-        ...
-        }
-      ```
+3. Register the plugin in the `packages/backend/src/plugins/catalog.ts` file. You can also configure a schedule in this step. However there are possible ways of configuration, such as:
+
+   - Configure a schedule inside the `app-config.yaml` file:
+
+     ```yaml
+     # app-config.yaml
+     catalog:
+       providers:
+         keycloakOrg:
+           default:
+             ...
+             schedule: # optional; same options as in TaskScheduleDefinition
+               # supports cron, ISO duration, "human duration" as used in code
+               frequency: { minutes: 1 }
+               # supports ISO duration, "human duration" as used in code
+               timeout: { minutes: 1 }
+               initialDelay: { seconds: 15 }
+     ```
+
+     Use the configured scheduler as follows:
+
+     ```diff
+     // packages/backend/src/plugins/catalog.ts
+     + import { KeycloakOrgEntityProvider } from '@janus-idp/backstage-plugin-keycloak-backend';
+
+       export default async function createPlugin(
+         env: PluginEnvironment,
+       ): Promise<Router> {
+         const builder = await CatalogBuilder.create(env);
+
+     +   builder.addEntityProvider(
+     +     KeycloakOrgEntityProvider.fromConfig(env.config, {
+     +       id: 'development',
+     +       logger: env.logger,
+     +       scheduler: env.scheduler,
+     +     }),
+     +   )
+       ...
+       }
+     ```
+
+   - Add a schedule directly inside the `packages/backend/src/plugins/catalog.ts` file as follows:
+
+     ```diff
+     // packages/backend/src/plugins/catalog.ts
+     + import { KeycloakOrgEntityProvider } from '@janus-idp/backstage-plugin-keycloak-backend';
+
+       export default async function createPlugin(
+         env: PluginEnvironment,
+       ): Promise<Router> {
+         const builder = await CatalogBuilder.create(env);
+
+     +   builder.addEntityProvider(
+     +     KeycloakOrgEntityProvider.fromConfig(env.config, {
+     +       id: 'development',
+     +       logger: env.logger,
+     +       schedule: env.scheduler.createScheduledTaskRunner({
+     +         frequency: { minutes: 1 },
+     +         timeout: { minutes: 1 },
+     +         initialDelay: { seconds: 15 }
+     +       }),
+     +     }),
+     +   )
+       ...
+       }
+     ```
```

Comment:
I'd consider breaking this into a couple of steps instead, i.e. the required step of registering the plugin, and then the option step of configuring the scheduler

Reply:
This action is already in steps: required step is in numbered list and option steps are in bulleted list.